invoice/client_test.go

```diff
@@ -502,7 +502,7 @@ func TestAllInvoicesScenarios(t *testing.T) {
 		t.Errorf("Updated invoice paid status %v does not match expected true\n", targetInvoiceUpdated.Paid)
 	}
 
-	_, err = plan.Del(planParams.ID)
+	_, err = plan.Del(planParams.ID, nil)
```

Comment:
This is unrelated to the PR, but I had to make this modification otherwise the test wouldn't compile. I guess it's related to #425.